src/components/input-dropdown/input-dropdown.component.less

```diff
@@ -9,6 +9,12 @@ ux-input-dropdown {
         white-space: nowrap;
         overflow: hidden;
     }
+
+    &.ux-select-disabled .ux-select-container .ux-select-icons {
+        .ux-select-icon, .ux-select-icon {
```

Comment:
These selectors are identical.

The selector here is much too specific, you can probably use 
```
&.ux-select-disabled .ux-select-icon
```

Reply:
These selectors are specific but they need to be for the icons, when I use &.ux-select-disabled .ux-select-icon then the icon doesn't have the correct disabled state colour so it is necessary to have all of the selectors included